brainscore_language/models/huggingface.py

```diff
@@ -77,13 +83,12 @@ def digest_text(self, text: Union[str, List[str]]) -> Dict[str, DataAssembly]:
             #  Maybe we need a StimulusSet class with metadata instead of just text strings after all?
             'stimulus_id': ('presentation', [0] if isinstance(text, str) else np.arange(len(text))),
         }
+
         if self.behavioral_task:
-            logits = base_output.logits
-            pred_id = torch.argmax(logits, axis=2).squeeze()
-            last_model_token_inference = pred_id[-1].tolist()
-            next_word = self.tokenizer.decode(last_model_token_inference)
+            behavioral_output = self.run_behavioral_experiment(text=text,
+                                                    base_output= base_output)
```

Comment:
As above (+ `text` isn't actually used)

```suggestion
            behavioral_output = self.output_to_behavior(base_output= base_output)
```